src/main/java/com/patrickjones/GsonPrinter.java

```diff
@@ -0,0 +1,36 @@
+/*Add a new class called GsonPrinter with a main() method that creates a new BancAccount object
+and then uses Gson to print it out as JSON.
+result; {"accountName":"Ryan's Account","balance":10000,"accountNumber":2}*/
+package com.patrickjones;
+
+import com.google.gson.Gson;
+import com.patrickjones.models.BancAccount;
+
+public class GsonPrinter {
+
+    public static void main(String[] args) {
+
+        BancAccount ryan = Object();
+        Gson gson = new Gson();
+        String json = gson.toJson(ryan);
+        System.out.println(json);
+
+    }
+
+    private static BancAccount Object() {
```

Comment:
Let's rename this method to `createBankAccount()` so it's a little more self-descriptive.